doc/classes/Tween.xml

```diff
@@ -219,8 +219,14 @@
 			<return type="Tween" />
 			<param index="0" name="ease" type="int" enum="Tween.EaseType" />
 			<description>
-				Sets the default ease type for [PropertyTweener]s and [MethodTweener]s animated by this [Tween].
-				If not specified, the default value is [constant EASE_IN_OUT].
+				Sets the default ease type for [PropertyTweener]s and [MethodTweener]s appended after this method.
+				Before this is set, the default ease type is [constant EASE_IN_OUT].
```

Comment:
My main concern with the phrasing of "If not specified" or "If this method is not called" is that, at least to me, it implies that the order of calling it doesn't matter for setting the default ease type (and transition type).

For example, in this snippet of code:
```gdscript
var tw := create_tween()
tw.tween_property(self, "position:x", 50, 1.0)
tw.set_ease(Tween.EASE_IN)
```
With the phrasing of
> If this method is not called, the default ease type is `EASE_IN_OUT`.

I would expect that the single property Tweener there would adopt the newly-set default of `EASE_IN` once it is run, not that it would still have `EASE_IN_OUT` because the method wasn't called after the Tweener was created. Granted, the sentence prior ends with "appended after this method", which explains the order-dependent nature, but this following sentence seems like it would suggest the order isn't dependent (that it can be called at *any* time to change the default ease type, which isn't exactly the case).

Reply:
Not a fan of the "before this is set" because of the strict time dependency, and implication that there is an absolute need to call this method, but I see what you mean.
```suggestion
				Before this method is called, the default ease type is [constant EASE_IN_OUT].
```

Reply:
Thanks, this phrasing sounds good to me!

I may file a proposal to change how default transition and ease types are handled, such that there isn't the time/order dependency. If it managed to go through, I suppose it'd make this PR obsolete, but at least in the interim it will be helpful 😅

Reply:
I think the current behavior is fine and expected. The order of operations for Tween & Co. matters a lot.